modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -36,6 +36,7 @@ def adapter
       def bgs_service_response
         person = BGS::People::Request.new.find_person_by_participant_id(user: current_user)
         if person.response.blank?
+          # this does reveal that some users have blank icns, which is suprising to me, but this is meaningless unless we're trying to use this info
           Rails.logger.info('Mobile Payment History Person not found for user icn: ',
```

Comment:
Category: i am here with id.
Action: this is interesting because it does incidentally reveal that icns can be nil. I'm not sure what to make of that info. Ask around about icn issue. Remove logging.